docs/index.md

```diff
@@ -39,32 +39,21 @@ to you with the software itself.
 
 - **Manufacturer Name**: {{device_manufacturer}}
 - **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
-- **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
+- **Software as a Medical Device Identification**: {{device_name}}
+  {{device_version}}
 
 ### Difference to the open source version
 
-This device corresponds to the software downloadable directly from
-<https://radiotherapy.ai/> (Regulated Medical Device). It does **not**
```

Comment:
The Web sites should be configuration items

Reply:
Yup, that was actually part of this change. This section (in red) is the text that has been removed. The updated line has the following:

https://github.com/OpenSaMD/OpenSaMD/blob/ccf95652a5a2334fee9f97558db77567efb57eaa/docs/index.md?plain=1#L48-L50

![image](https://user-images.githubusercontent.com/6559099/205761797-5ff0d13a-8c5d-4587-a8fe-227c0264c9aa.png)